Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -34,10 +34,9 @@ void NANDImporter::ImportNANDBin(const std::string& path_to_bin,
   if (!ReadNANDBin(path_to_bin, get_otp_dump_path))
     return;
 
-  const std::string nand_root = File::GetUserPath(D_WIIROOT_IDX);
+  std::string nand_root = File::GetUserPath(D_WIIROOT_IDX);
+  nand_root.pop_back();  // remove trailing path separator
```

Comment:
IIRC it doesn't matter if this path has a trailing separator, the only piece of code that does (or did) care is to make debug logs a little nicer to look at. So I don't think this line is needed but feel free to do it anyways.

PS: Annoyed by my old code in either case, I made a quick refactor that completely removes the weird `_length` variable (and a few other things) at https://github.com/Starsam80/dolphin/commit/8911c3bb42ba8abaa6beffbe7924b0dd622cc77e which I might make a PR for if I ever get enough energy.